core/server/services/routing/controllers/collection.js

```diff
@@ -48,7 +48,7 @@ module.exports = function collectionController(req, res, next) {
 
```

Comment:
This commit refs a different issue. We will merge all commits into master. No need to outsource the commit IMO.